### PR TITLE
Bump ES startup time from 2 minutes to 4 minutes

### PR DESCRIPTION
**Pull request by tgjonestx**
Fixes CC-1356

### CODE REVIEW

**Comment by tgjonestx** [comment]
Retest this

**Comment by tgjonestx** [comment]
Retest this

**Comment by tgjonestx** [comment]
Retest this

File: isvcs/container.go [hunk 1]
```diff
@@ -662,7 +662,13 @@ func (svc *IService) startupHealthcheck() <-chan error {
 				currentTime := time.Now()
 				result = svc.runCheckOrTimeout(checkDefinition)
 				svc.setHealthStatus(result, currentTime.Unix())
-				if result == nil || time.Since(startCheck).Seconds() > svc.StartupTimeout.Seconds() {
+				elapsed := time.Since(startCheck)
+				if result == nil {
+					glog.Infof("Verified health status of %s after %s", svc.Name, elapsed)
+					break
+				} else if elapsed.Seconds() > svc.StartupTimeout.Seconds() {
+					glog.Errorf("Could not verified health status of %s after %s. Last health check returned %#v",
```
Comment (alapidas) [inline isvcs/container.go]: probably mean to say 'verify' instead of 'verified' here

**Pull request merged**